mods/ts/rules/civilian-structures.yaml

```diff
@@ -722,6 +722,14 @@ CAARAY:
 		HP: 400
 	RenderSprites:
 		Palette: player
+	WithIdleOverlay@SATELLITE:
+		Sequence: idle-satellite
+	WithIdleOverlay@RADAR:
+		Sequence: idle-radar
+	WithIdleOverlay@SCANNER:
+		Sequence: idle-scanner
+	WithIdleOverlay@LIGHT:
+		Sequence: idle-light
```

Comment:
This is a Blinkenlight, so shouldn't it use one of the effect-ignore-lighting(-alpha?) palettes?

Reply:
Looked at other buildings for reference and couldn't find `effect-ignore-lighting` palettes.

Reply:
@pchote My PR for that wasn't merged because we don't have a solution for player palettes yet (neither alpha nor palette modifiers work as intended on them).

Reply:
aha, I see.  Do we have an issue # for that?

Reply:
...although I wouldn't have thought that most of the lights would be using a player specific palette?

Reply:
Most wouldn't, but some do (CYard top lights, refinery lights).

However, I just remembered what the issue was:

Original TS simply left a few hard-coded palette indices of the player palette unmodified to make them stay bright on darker maps, the `ignore-lighting` palette approach wouldn't work perfectly on some sprites (because it would 'lighten up' indices that should actually follow the global lighting) and would become redundant once we implement some `IgnoreIndices` feature.

Reply:
Right, I remember that now. #13046.